addons/hr_holidays/static/src/js/dist/time_off_calendar.dev.js

```diff
@@ -0,0 +1,188 @@
+"use strict";
+
+odoo.define('hr_holidays.dashboard.view_custo', function (require) {
```

Comment:
```suggestion
odoo.define('hr_holidays.dashboard.view_custo', function (require) {
```

Reply:
File removed, idk why it has been commited